src/Config/Parser/AnnotationParser.php

```diff
@@ -490,16 +490,17 @@ private static function unionAnnotationToGQLConfiguration(string $className, GQL
         if ($unionAnnotation->resolveType) {
             $unionConfiguration['resolveType'] = self::formatExpression($unionAnnotation->resolveType);
         } else {
-            if (isset($methods['resolveType'])) {
-                $method = $methods['resolveType']['method'];
-                if ($method->isStatic() && $method->isPublic()) {
-                    $unionConfiguration['resolveType'] = self::formatExpression(\sprintf("@=call('%s::%s', [service('overblog_graphql.type_resolver'), value], true)", self::formatNamespaceForExpression($className), 'resolveType'));
-                } else {
-                    throw new InvalidArgumentException(\sprintf('The "resolveType()" method on class must be static and public. Or you must define a "resolveType" attribute on the @Union annotation.'));
-                }
-            } else {
-                throw new InvalidArgumentException(\sprintf('The annotation @Union has no "resolveType" attribute and the related class has no "resolveType()" public static method. You need to define of them.'));
+            if (!isset($methods['resolveType'])) {
+              throw new InvalidArgumentException(\sprintf('The annotation @Union has no "resolveType" attribute and the related class has no "resolveType()" public static method. You need to define of them.'));
```

Comment:
```suggestion
                throw new InvalidArgumentException(\sprintf('The annotation @Union has no "resolveType" attribute and the related class has no "resolveType()" public static method. You need to define of them.'));
```

Fix CS: add 2 more spaces before `throw`